default/stringtables/en.txt

```diff
@@ -6674,17 +6674,27 @@ GREETINGS_GUIDE_TEXT
 
 FreeOrion is a free, open source, turn-based space empire and galactic conquest (4X) computer game being designed and built by the FreeOrion project. FreeOrion is inspired by the tradition of the Master of Orion games, but is not a clone or remake of that series or any other game.
 
-Further information is available at freeorion.org
+Further information is available at http://www.freeorion.org
 '''
 
 QUICK_START_GUIDE_TITLE
 **Quick Start Guide**
 
 QUICK_START_GUIDE_TEXT
-'''This is a brief introduction to FreeOrion gameplay. A somewhat less brief Quick Play Guide is available at freeorion.org (current URL: <url  http://freeorion.org/index.php/V0.4_Quick_Play_Guide>freeorion.org/index.php/V0.4_Quick_Play_Guide</url> ).
+'''The Pedia is ment to assist with and inform about the many strange new features of this great game. Right-clicking on many items on the game screen will lead to specific Pedia articles. But first, lets have a look around.
```

Comment:
have another editing pass. this needs more work

Reply:
it should be fixed when all the other changes are final... until then, it doesn't matter